### PR TITLE
[CA-1306] A11y: Workflows pages

### PR DESCRIPTION
**Pull request by andrewmarcus**
This PR addresses accessibility issues on workflows pages.

## Summary of changes

* Marked up the TabBar widget as role=tablist
* Adjusted Prism code editor theme colors to increase contrast to 4.5:1
* Added a label to the snapshot selector input
* Added a "main" landmark surrounding the Workflows page contents
* Converted dashboard headers to actual h2 tags

## Notes and Questions

How do users actually navigate to this set of pages? I don't see it in the main menu, and the Workflows tab in the Workspaces section appears to not go here directly.

I've changed the Prism theme colors directly in the style.css file with `!important` overrides. However, it appears that Prism is based on a theme system and ships with a number of built-in themes, so perhaps it's also possible to define a custom theme? Would that be a better solution than what I did here?

How would I see an example of the code editor not being readonly? In every one of the handful of workflows I looked at on the dev server, nothing seemed to be editable...

### CODE REVIEW

**Review by s-rubenstein — APPROVED** [review]
This looks good to me. I believe you get to the workflow details page by running a workflow and then clicking into it. For the WDLViewer page, I believe it is when you go to import a workflow from the methods repository, but I am not certain.

File: src/style.css [hunk 1]
```diff
@@ -127,3 +127,27 @@ a {
   animation: progress-bar-stripes 1s linear infinite;
 }
 
+/* Prism theme overrides to increase contrast for accessibility */
```
Comment (andy7i) [inline src/style.css:130]: >I've changed the Prism theme colors directly in the style.css file with !important overrides. However, it appears that Prism is based on a theme system and ships with a number of built-in themes, so perhaps it's also possible to define a custom theme? Would that be a better solution than what I did here?


It looks like there is a way to create a custom theme (http://k88hudson.github.io/syntax-highlighting-theme-generator/www/) and there are some a11y-compliant themes (https://github.com/ericwbailey/a11y-syntax-highlighting) but I don't have a strong preference with any of these 3 approaches. Any thoughts from you or @s-rubenstein ?

**Review by andy7i — APPROVED** [review]
LGTM, did not do any manual testing. One question but not blocking

to answer your questions:

>How do users actually navigate to this set of pages? I don't see it in the main menu, and the Workflows tab in the Workspaces section appears to not go here directly.

I asked around and it sounds like the `Workflows` list is not a completed feature yet, and so it is not linked anywhere in Terra. I think you're right in that it can only be accessed by manually pointing the browser to the URL https://bvdp-saturn-dev.appspot.com/#workflows . Similarly, `WorkflowDetails` seems to only be used by the `Workflows` list page. 

>I've changed the Prism theme colors directly in the style.css file with !important overrides. However, it appears that Prism is based on a theme system and ships with a number of built-in themes, so perhaps it's also possible to define a custom theme? Would that be a better solution than what I did here?

Answered this in a code comment so it's easier to reply to.

>How would I see an example of the code editor not being readonly? In every one of the handful of workflows I looked at on the dev server, nothing seemed to be editable...

I didn't see anything for editing wdl workflows either. I think users would go to the firecloud method repo to make changes to a wdl.

File: src/style.css (hunk 1 above)
Comment (andrewmarcus) [reply src/style.css:130]: Hmm, I like that Eric Bailey one, and the fact that it has support for high-contrast mode. That's a feature we should be considering as well.

The main thing, though, if I switch to a different theme, is that the colors could change rather dramatically. I'm not sure anyone would notice or care, but that's definitely a thing to think about. In this PR I've just taken the existing colors and darkened them according to ANDI's recommendations to get them just barely to 4.5:1

Comment (s-rubenstein) [reply src/style.css:130]: Yeah, I think changing to a new theme would be a larger conversation with UX. There's a lot of consideration that has gone into our themes color design both from UX, but also likely from project partners. My guess is there are some theme requirements.

Comment (andrewmarcus) [reply src/style.css:130]: To be clear, we're just talking about the color scheme of the code editor, not the site theme overall.

Do you think the adjustments I made will be ok without a review from UX? The most obvious one is that the orange has turned into a sort of orangey-brown. The other color switches are pretty subtle.

Comment (zarsky-broad) [reply src/style.css:130]: Fwiw, I think we just... _picked one_.

Comment (zarsky-broad) [reply src/style.css:130]: (As such I think whatever you did is fine @andrewmarcus)

Comment (s-rubenstein) [reply src/style.css:130]: Ohhhhh… I didn't realize only the code editor. I think what you did is fine, and then we can re-evaluate if the workflows teams want to change it longer term.

Comment (andy7i) [reply src/style.css:130]: ok, that all sounds good, thanks for the input!

**Pull request merged**